all-spark-notebook/Dockerfile

```diff
@@ -22,19 +22,18 @@ RUN apt-get update && \
 USER $NB_UID
 
 # R packages
-RUN conda install --quiet --yes \
-    'r-base=4.0.3' \
+RUN mamba install --quiet --yes \
```

Comment:
This takes too much time with conda.
I replace the parts which were taking too long with conda on my machine.

Let's try mamba, it seems to work great.

Reply:
I'm very supportive of this transition, it has worked out great and made a significant difference in performance.

Reply:
On this point the last time we tried to switch to mamba we noticed differences on the size of images. It seemed that something was missing (in fact we had no clear explanation at the time) in R images. See [here](https://github.com/jupyter/docker-stacks/pull/1213#issuecomment-761517281) for the summary. I think we should check if we have the same behavior here before performing this change.